support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -533,7 +549,27 @@ function CheckoutComponent({ geoId }: Props) {
 				.then((response) => response.json())
 				.then((json) => json as StatusResponse);
 
-			processPayment(createSubscriptionResult, geoId);
+			const processPaymentResponse = await processPayment(
+				createSubscriptionResult,
+				geoId,
+			);
+			if (processPaymentResponse.status === 'success') {
+				const order = {
+					firstName: personalData.firstName,
+					price: price,
+					product: productId,
+					ratePlan: query.ratePlan,
+					paymentMethod: paymentMethod as string,
+				};
+				setThankYouOrder(order);
```

Comment:
Can see the session storage call with thankyou order object here and then going through the router to the thankyou page below. Nice!